### ISSUE TITLE
Question on RAT links & faulty implementation in version 2.28.3?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-app-android#5814]
## Avoid duplicates
* [X] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Question is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Question has not already been asked in another issue

## Your Question
I have a user on Twitter who attempted to scan the QR code of a test, which lead the app to show the error `Generated hash doesn't match QRCode hash`. The first attempt to scan the QR code with the app was made with version 2.28.3. Clicking on the link to register the test in the app, which was provided on addition to the QR code by the testing site, caused the app to crash.

After updating the app to version 3.0.1, registering the test by clicking on the link was successful. Scanning the QR code on version 3.0.1 now leads to the error that the test was already registered, but doesn't cause the app to show the `Generated hash doesn't match QRCode hash` error anymore.

Was there a faulty implementation in version 2.28.3 of the app?

The link provided by the testing site begins with `https://s.coronawarn.app/?v=1#` (note that there is a / after .app) & the link in the QR code begins with `https://s.coronawarn.app?v=1#` (note that there is no / after .app). Is this difference normal, or could it have caused the `Generated hash doesn't match QRCode hash` error the user saw? If yes, why doesn't the app still show the error on version 3.0.1?

**Comment by dsarkar** [comment]
@Ein-Tim We will try to get an answer, stand by, please.

**Comment by Ein-Tim** [comment]
@dsarkar

Were you able to get an answer to this?

**Comment by dsarkar** [comment]
@Ein-Tim Indeed, the slash should not lead to "Generated hash doesn't match QRCode hash". Apparently the provider generated a erroneous hash in the QR-code.

**Comment by Ein-Tim** [comment]
@dsarkar 

The rest of the link @ the QR code is identical...
I can send the information to @thomasaugsten via mail if wanted.

**Comment by dsarkar** [comment]
@Ein-Tim I will check with him. Thanks for the offer.

**Comment by Ein-Tim** [comment]
As the CWA project went into ramp-down mode, I'm now closing this issue, as I'm not interested in the answer to my question any longer.
